### ISSUE TITLE
Issue when training on a custom dataset.

### ISSUE THREAD

**Issue by vineeth2309** [nianticlabs/monodepth2#305]
I have collected my own stereo dataset, and have made some changes to the dataloader provided in the repository to load the data in the format I needed. The relative motion between the frames is very low, and therefore i am using frames [-8, 0, +8] to match the motion of the kitti data. During training, the model loss fluctuates in the same range of around 0.026 - 0.03. On inference , the model is able to predict the relative depth, however it is unable to predict a similar metric depth that I obtain from the pretrained model. Moreover the smoothness from the pretrained model seems to be quite different to the model I have trained. I have verified the loss functions, to double check that there is no error. I have attached the results from pretraining, and from the model I have trained.
![custom_train](https://user-images.githubusercontent.com/19423039/104682038-ccff1480-5719-11eb-8a08-a6a9c42972f4.png)
![pretrained](https://user-images.githubusercontent.com/19423039/104682096-eef89700-5719-11eb-823b-e021ade58b84.png)

**Comment by mrharicot** [comment]
Hi,

It is difficult to know what is wrong here but the depthmap looks very strange.
Could you share input samples from the training data? (-8, 0, +8 and stereo)

**Comment by vineeth2309** [comment]
I seem to have fixed the issue. I had made some modifications to the loss function and it seems like that was causing the issue.

**Comment by buaachen1993** [comment]
> I seem to have fixed the issue. I had made some modifications to the loss function and it seems like that was causing the issue.

hi， @vineeth2309 I meet the same problem using my own data, the smoothness is very strange, would you like to share what kind of modifications you made that fix this problem? Thanks a lot!
